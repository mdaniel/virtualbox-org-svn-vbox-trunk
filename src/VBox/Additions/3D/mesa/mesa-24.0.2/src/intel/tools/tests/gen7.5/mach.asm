mach(8)         g19<1>UD        g17<8,8,1>UD    0xaaaaaaabUD    { align1 1Q AccWrEnable };
mach(8)         g10<1>D         g17<8,8,1>D     1431655766D     { align1 1Q AccWrEnable };
mach(8)         g87<1>UD        g84<8,8,1>UD    0xaaaaaaabUD    { align1 2Q AccWrEnable };
mach(8)         g95<1>D         g84<8,8,1>D     1431655766D     { align1 2Q AccWrEnable };
mach(8)         null<1>D        g1<0>.xyzzD     g1<0>.wD        { align16 1Q AccWrEnable };
mach(8)         g12<1>UD        g9<4>UD         g11<4>UD        { align16 1Q AccWrEnable };
mach(8)         g12<1>UD        g4<8,8,1>UD     g8<8,8,1>UD     { align1 1Q AccWrEnable };
mach(8)         g21<1>UD        g5<8,8,1>UD     g13<8,8,1>UD    { align1 2Q AccWrEnable };
mach(8)         g15<1>D         g12<4>D         g14<4>D         { align16 1Q AccWrEnable };
mach(8)         g7<1>.xUD       g5<4>.xUD       0xaaaaaaabUD    { align16 1Q AccWrEnable };
mach(8)         g15<1>.xD       g5<4>.xD        1431655766D     { align16 1Q AccWrEnable };
mach(8)         null<1>D        g1<4>.xD        741092396D      { align16 1Q AccWrEnable };
mach(8)         g13<1>D         g5<8,8,1>D      g9<8,8,1>D      { align1 1Q AccWrEnable };
mach(8)         g22<1>D         g6<8,8,1>D      g14<8,8,1>D     { align1 2Q AccWrEnable };
