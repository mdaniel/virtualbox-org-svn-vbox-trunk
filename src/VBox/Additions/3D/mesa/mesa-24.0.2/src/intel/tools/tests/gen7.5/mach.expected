49 00 60 10 21 0c 60 22 20 02 8d 00 ab aa aa aa
49 00 60 10 a5 1c 40 21 20 02 8d 00 56 55 55 55
49 10 60 10 21 0c e0 2a 80 0a 8d 00 ab aa aa aa
49 10 60 10 a5 1c e0 2b 80 0a 8d 00 56 55 55 55
49 01 60 10 a4 14 0f 20 24 00 0a 00 2f 00 0f 00
49 01 60 10 21 04 8f 21 24 01 6e 00 64 01 6e 00
49 00 60 10 21 04 80 21 80 00 8d 00 00 01 8d 00
49 10 60 10 21 04 a0 22 a0 00 8d 00 a0 01 8d 00
49 01 60 10 a5 14 ef 21 84 01 6e 00 c4 01 6e 00
49 01 60 10 21 0c e1 20 a0 00 60 00 ab aa aa aa
49 01 60 10 a5 1c e1 21 a0 00 60 00 56 55 55 55
49 01 60 10 a4 1c 0f 20 20 00 60 00 2c 2c 2c 2c
49 00 60 10 a5 14 a0 21 a0 00 8d 00 20 01 8d 00
49 10 60 10 a5 14 c0 22 c0 00 8d 00 c0 01 8d 00
